src/msg/async/rdma/RDMAStack.h

```diff
@@ -189,6 +189,7 @@ class RDMAConnectedSocketImpl : public ConnectedSocketImpl {
   std::vector<ibv_wc> wc;
   bool is_server;
   EventCallbackRef con_handler;
+  EventCallbackRef con_init_handler;
```

Comment:
why not use a `unique_ptr<EventCallback>`?

Reply:
s/con_init_handler/established_handler/

and add a commit to s/con_handler/read_handler/

Reply:
> why not use a `unique_ptr<EventCallback>`?


> why not use a `unique_ptr<EventCallback>`?
1.  keep it the style  with the old code; 
2.  we don't need to use a unique_ptr<EventCallback>,  I will free con_init_handler in `cleanup()`.